build.sbt

```diff
@@ -219,18 +219,18 @@ lazy val unit = project
     testkit
   )
 
-lazy val website = project
-  .enablePlugins(MicrositesPlugin)
-  .enablePlugins(ScalaUnidocPlugin)
+lazy val docs = project
+  .in(file("scalafix-docs"))
   .settings(
+    noMima,
+    skip in publish := true,
+    moduleName := "scalafix-docs",
+    mainClass.in(Compile) := Some("docs.Main"),
     scalaVersion := scala212,
-    noPublish,
-    websiteSettings,
-    unidocSettings,
-    libraryDependencies += "com.geirsson" %% "metaconfig-docs" % metaconfigV,
-    unidocProjectFilter in (ScalaUnidoc, unidoc) := inProjects(
-      testkit,
-      core
+    libraryDependencies ++= List(
+      "com.geirsson" %% "metaconfig-docs" % metaconfigV,
+      "com.geirsson" % "mdoc" % "0.4.0" cross CrossVersion.full,
```

Comment:
do you need the explicit `mdoc` dependency even when using the `DocusaursPlugin`?